scripts/development.sh

```diff
@@ -122,7 +122,8 @@ function dco::helm_install() {
     --set image.registry="" \
     --set image.repository="$IMAGE_NAME" \
     --set image.tag="$latest_tag" \
-    --set config.logDevelopmentMode=true
+    --set config.logDevelopmentMode=true \
+    --set istio.enabled=true
```

Comment:
I guess we want to "default" to using Istio? This doesn't seem like a crazy default, but I want to make sure this was your intention.